src/decomposition.jl

```diff
@@ -85,10 +92,36 @@ function get_indices_of_obj_in_partition(
     return tuple
 end
 
-get_indices_of_obj_in_partition(obj_ref::JuMP.ConstraintRef, _, _) = ()
-get_indices_of_obj_in_partition(obj_ref::JuMP.VariableRef, _, _) = ()
-get_indices_of_obj_in_partition(obj_ref::Array{<:JuMP.VariableRef, N}, _, _) where N = ntuple(i -> Colon(), N)
-get_indices_of_obj_in_partition(obj_ref::Array{<:JuMP.ConstraintRef, N}, _, _) where N = ntuple(i -> Colon(), N)
+function get_indices_of_obj_in_partition(
```

Comment:
this is also a bit obscure. Can you add a comment explaining what it does?
I searches all the indices in `obj_ref` that corresponds to `dec_axes_val` in some way? Also, what is dec_axes_val? you type these functions to improve readability?